src/risclog/sqlalchemy/db.py

```diff
@@ -92,17 +96,21 @@ def get_database(testing=False):
 @zope.interface.implementer(risclog.sqlalchemy.interfaces.IDatabase)
 class Database:
 
-    def __init__(self, testing=False):
+    def __init__(
+        self, testing=False, keep_session=False, expire_on_commit=False
+    ):
         assert zope.component.queryUtility(
             risclog.sqlalchemy.interfaces.IDatabase) is None, \
             'Cannot create Database twice, use `.get_database()` to get '\
             'the instance.'
         self._engines = {}
         self.testing = testing
         self.session_factory = sqlalchemy.orm.scoped_session(
-            sqlalchemy.orm.sessionmaker(class_=RoutingSession))
+            sqlalchemy.orm.sessionmaker(
+                class_=RoutingSession,
+                expire_on_commit=expire_on_commit))
```

Comment:
The [SQLAlchemy documentation](https://docs.sqlalchemy.org/en/14/orm/session_api.html?highlight=sessionmaker#sqlalchemy.orm.sessionmaker) says that the default value of `expire_on_commit` is `True`. I would like to keep that also as a default value here, as it might have unintended effects in downstream projects.